cluster/examples/kubernetes/ceph/csi/rbd/storageclass-test.yaml

```diff
@@ -33,6 +33,20 @@ parameters:
     #dataPool: ec-data-pool
     pool: replicapool
 
+    # (optional) mapOptions is a comma-separated list of map options.
```

Comment:
I'd rather keep the `-test.yaml` examples simple than include all the options. How about we revert this file?

Reply:
done

Reply:
Do you still need to push?